mint.json

```diff
@@ -314,6 +314,27 @@
             "api-reference/admin/user-management/delete-a-user-by-id"
           ]
         },
+        {
+          "group": "Email Management",
+          "pages": [
+            "api-reference/admin/email-management/get-a-list-of-emails-by-user-id",
```

Comment:
for some weird reason when i click on this, the whole page reloads, not sure why this is happening. Everything seems right imo.
<img width="347" alt="Screenshot 2024-01-23 at 11 42 15 AM" src="https://github.com/teamhanko/docs/assets/62984427/0cfac9ae-bc0b-4902-a642-8c274eeb8da5">

Reply:
Seems to be a path length problem in mintlify. I shortened the path be removing the by user id and et voila.. it will work.